sdk/eventhub/azure-eventhubs/setup.py

```diff
@@ -62,7 +62,7 @@
         "tests",
         "tests.asynctests"]),
     install_requires=[
-        'uamqp~=1.1.0',
+        'uamqp~=1.2.3',
```

Comment:
Does this version include the socket timeout fix that was causing the "~30 min send blackout" issue?

Reply:
I believe so, if I'm reading the latest in the email thread properly.  Uamqp 1.2.3 + the seconds->milliseconds conversion in EH proper should address the sending timeout failure.

@yunhaoling feel free to shout if I misinterpreted that thread or failures.

Reply:
Just adding more context here: It's the receive blackout and simply introducing idle_timeout would bring new issues.
Need more efforts into the blackout issue.